sources/packages/web/src/types/contracts/students/ApplicationContract.ts

```diff
@@ -0,0 +1,50 @@
+/**
+ * Possible status of a Student Application.
+ */
+export enum ApplicationStatus {
+  /**
+   * The application is in draft state and is not Submitted
+   */
+  draft = "Draft",
+  /**
+   * The application has been submitted by the application
```

Comment:
Please change it to "The application has been submitted by the student"

Reply:
I literally copy-pasted your interface in API :). sure, will do